File: Downloads/StepCounterApp_anu-master_gitlab/StepCounterApp_anu-master_gitlab/app/src/main/java/com/sd/src/stepcounterapp/model/wallet/TokenModel.java

package com.sd.src.stepcounterapp.model.wallet;


@SuppressWarnings("unused")
public class TokenModel {

    private Data data;
    private String message;
    private long status;
    
    public Data getData() {
        return data;
    }
    
    public void setData(Data data) {
        this.data = data;
    }
    
    public String getMessage() {
        return message;
    }
    
    public void setMessage(String message) {
        this.message = message;
    }
    
    public long getStatus() {
        return status;
    }
    
    public void setStatus(long status) {
        this.status = status;
    }
}

File: app/src/main/java/com/sd/src/stepcounterapp/model/wallet/TokenModel.kt
package com.sd.src.stepcounterapp.model.wallet

import com.sd.src.stepcounterapp.model.BaseModel

class TokenModel : BaseModel() {
    /**
     * data : {"totalSteps":111,"totalEarntokens":0}
     */

    var data: DataBean? = null
    class DataBean {
        /**
         * totalSteps : 111
         * totalEarntokens : 0
         */
        var totalSteps: Int = 0
        var totalEarntokens: Int = 0
    }
}
